dapps/react-dapp-v2/src/components/Header.tsx

```diff
@@ -85,6 +86,9 @@ const Header = (props: HeaderProps) => {
             <Button outline color="red" onClick={disconnect}>
               {"Disconnect"}
             </Button>
+            <Button outline color="red" onClick={emit}>
```

Comment:
Would make this one black like ping and move it up one so we keep Disconnect on the furthest right (see the preview deployment)